airflow/www/templates/appbuilder/navbar.html

```diff
@@ -50,6 +50,16 @@
         {% include 'appbuilder/navbar_menu.html' %}
       </ul>
       <ul class="nav navbar-nav navbar-right">
+        <li class="active">
+          <div style="padding: 20px; cursor: pointer" id="themeToggleButton">
```

Comment:
Let's add a title and aria-label saying something like "Toggle dark mode (experimental)"

Reply:
I've added a title and aria-label to that element:)